pkgs/servers/home-assistant/custom-components/rct_power/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildHomeAssistantComponent,
+  rctclient,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "weltenwort";
+  domain = "rct_power";
+  version = "v0.14.1";
+
+  src = fetchFromGitHub {
+    owner = "weltenwort";
+    repo = "home-assistant-rct-power-integration";
+    rev = "refs/tags/${version}";
+    hash = "sha256-wM66MyRhBsMfUr+KlqV4jSuXcnKfW0fkbDAyuU2crsc=";
+  };
+
+  propagatedBuildInputs = [
```

```suggestion
  dependencies = [
```

This is the correct lingo for python. It is merged into `propagatedBuildInputs`, but this is easier to understand for people who are familiar with python.